src/specs/VideoNativeComponent.ts

```diff
@@ -266,12 +267,12 @@ type OnReceiveAdEventData = Readonly<{
 
 export type OnVideoErrorData = Readonly<{
   error: Readonly<{
-    errorString?: string; // android
+    errorString?: string; // android | web
```

Comment:
these changes will bring conflicts ... if you need to resolve it, you can skip these comments I think